src/main/java/com/squareup/catalog/demo/Main.java

```diff
@@ -124,7 +130,9 @@ void processArgs(String[] args) {
     // Process arguments associated with the example.
     String accessToken = null;
```

Comment:
the README mentions `export SQPAT={{ YOUR NEW ACCESS TOKEN }}`, can we set the default value to the envar if it's set?